### ISSUE TITLE
Notification shows wrong title and onClick does not work

### ISSUE THREAD

**Issue by LukeOwncloud** [clementine-player/Android-Remote#112]
***How to reproduce***

Choose/play a title, minimize Android-Remote, switch off the screen.
Wait and listen for a few songs and switch screen on again.

Notification shows wrong title. Often the one which was first selected.

Now click of the notification item: Item does away but Android-Remote does not launch.

Version: 10.1

**Comment by amuttsch** [comment]
Please try again with the latest build and check if the issue is still present.

**Comment by LukeOwncloud** [comment]
Sounds great! https://github.com/clementine-player/Android-Remote/releases is still at version 10.1. Where can I download the latest apk?

**Comment by amuttsch** [comment]
You can download the latest version here: http://builds.clementine-player.org/android/

Note: You have to uninstall the play store version first, because the buildbot builds are signed differently.

**Comment by LukeOwncloud** [comment]
The latest version (http://builds.clementine-player.org/android/ClementineRemote-release-v10.1-102-g6951b6d.apk) tells me that I need to upgrade Clementine (minimum 1.2.1) even though 1.2.3 is installed.

**Comment by amuttsch** [comment]
Yeah, I haven't updated the string to the unreleased 1.3 yet. You have to install a hourly Clementine build from here: http://builds.clementine-player.org/

**Comment by rollopack** [comment]
The title problem persist with the latest release v10.1-199-ge3f1d92.apk

**Comment by sean-abbott** [comment]
I also have this problem, although have not tried anything other than full releases.
